fedora_elections/elections.py

```diff
@@ -379,38 +371,30 @@ def election_results_text(election_alias):
         stats=stats,
     )
 
-def process_vote(candidates, election, votes, revote):
+def process_vote(candidates, election, votes, revote, cand_name=None, value=None):
     for index in range(len(candidates)):
         candidate = candidates[index]
         if revote:
             vote = votes[index]
-            vote.value = int(candidate.data)
+            if value == 1:
```

Comment:
What if one day ``value > 1``, simplify it and use either: ``if value`` or maybe more robust: ``if value is not None``